### ISSUE TITLE
Feature Request: Lockscreen view stays

### ISSUE THREAD

**Issue by BDudely** [peterhajas/MobileNotifier#213]
I have a suggestion to add an option in the settings pane to have it so that MN automatically displays the notifications on the lockscreen, instead of you having to tap the icon for them to show

**Comment by Kaizersoze** [comment]
+1
And if this option is set to TRUE, then don't show the button on which we must tap to show notifications but only the notifications list. Also have the ability to sort notifications by priority.
1) Calendar reminders and invites, 2) missed calls, 3) voicemails, 4) SMS / MMS, 5) Mails, 6) Apps Notifications.
In a next step, make settings for this priorities.

**Comment by peterhajas** [comment]
BDudely, you're suggesting to just leave that list expanded?

**Comment by Kaizersoze** [comment]
Not only leave the list expanded but also remove the button "missed notifications" on which we tap to expand the list

**Comment by BDudely** [comment]
Yes the list is always expanded/showing

**Comment by LMD75** [comment]
I totally agree with this. I think it would look more professional if the 'Missed notifications' button is removed and you just have the list of notifications displayed /expanded.

**Comment by BDudely** [comment]
Any updates on this?
